### ISSUE TITLE
"black" requirement causes unnecessary version conflicts

### ISSUE THREAD

**Issue by pappasam** [bashtage/sphinx-material#17]
tl;dr please remove "black" from "install_requires" in `setup.py`.

"black" is a development dependency for this project. Unfortunately, it is being treated as a runtime dependency in setup.py and pinned to version 19.3b0. My project uses black version 19.10b0 as a development dependency, which makes my library's development environment unnecessarily incompatible with this sphinx theme.

**Comment by bashtage** [comment]
This is fixed in most recent release.
